blocks/footer/footer.css

```diff
@@ -1,14 +1,312 @@
+  padding: 1.25rem 3.125rem;
+  width: 100%;
+  margin-bottom: -3.75rem !important;
+}
+
+.footer.block .section .footer-top-container {
+  padding-top: 0;
+}
+
+.footer.block .footer-top p {
+  font-size: 0.9375rem;
+  margin-bottom: 0.5rem !important;
 }
 
-footer .footer {
-  max-width: 1200px;
+.footer.block .footer-top .col-1  img {
+    flex: 0 0 auto;
+    border: 0;
+    vertical-align: middle;
+    max-width: 100%;
+    width: 17.19rem;
+    height: 2.375rem;
+}
+
+.footer.block .footer-top .col-2 {
+  flex: 0 0 auto;
+  width: 50%;
+  text-align: center;
+  position: relative;
+  margin-bottom: 0;
+}
+
+.footer.block .footer-top .col-2::before {
+  height: 6rem;
+  background: var(--veritcal-line-footer-top-gray);
+  width: 0.0625rem;
+  content: "";
+  position: absolute;
+  left: 1.25rem;
+  top: 0;
+  bottom: 0;
   margin: auto;
 }
 
-footer .footer p {
-  margin: 0;
+.footer.block .footer-top .col-2::after {
+  height: 6rem;
+  background: var(--veritcal-line-footer-top-gray);
+  width: 0.0625rem;
+  content: "";
+  position: absolute;
+  right: 1.25rem;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.footer.block .footer-disclaimer p {
+  color: var(--color-footer-gray);
+  font-size: 0.6875rem;
+  line-height: 0.8125rem;
+}
+
+.footer.block .footer-top .col-3 {
+  flex: 0 0 auto;
+  width: 25%;
+}
+
+.footer.block .footer-top .col-3 p {
+  text-align: left;
+  padding-left: 1rem;
+}
+
+.footer.block .footer-links {
+  background: var(--link-color-dark-gray);
+  padding: 6.875rem 0 0;
+} 
+
+.footer.block .footer-links .links-title {
+  font-size: var(--body-font-size-xs);
+  line-height: 130%;
+  color: var(--color-off-white);
+  font-weight: 500;
+  display: block;
+  margin-bottom: 0.75rem;
+  padding-top: 1rem;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links .links-title {
+    font-size: var(--body-font-size-m);
+    line-height: 130%;
+    font-weight: 500;
+    margin-bottom: 0.75rem;
+  }
+}
+
+.footer.block .footer-links a {
+  color: var(--color-footer-gray);
+  transition: 0.3s;
+  line-height: 138%;
+  width: 100%;
+  max-width: 100%;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 12px;
+  padding: 0.3125rem 0;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links a {
+    font-size: 0.8125rem;
+  }
+}
+
+.footer.block .footer-links a > u {
+  text-decoration: none;
+}
+
+.footer.block .footer-disclaimer a {
+  color: var(--link-color-orange);
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 0.6875rem;
+  transition: 0.3s;
+}
+
+.footer.block .footer-links a:hover {
+  color: var(--color-off-white);
+}
+
+.footer.block .footer-links .links-column {
+  flex: 0 0 auto;
+  width: 50%;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links .links-column {
+    flex: 0 0 auto;
+    width: 16.6667%;
+    padding-left: 1rem;
+  }
+}
+
+.footer.block .footer-links .links-column > div {
+  margin: 0 0 1.25rem
+}
+
+@media (width < 37.5rem) {
```

Comment:
We should avoid `<` queries and keep the styles "mobile-first" i.e. the default styles are for mobile and wide screen styles are to be specified via media queries.

Reply:
+1

Reply:
done